src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,106 @@
+                PreparedStatement statement =
+                        connection.prepareStatement(createQuery, Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, driver.getName());
+            statement.setString(2, driver.getLicenseNumber());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                driver.setId(resultSet.getObject(1, Long.class));
+            }
+            return driver;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert a driver into DB"
+            + driver.getName() + " " + driver.getLicenseNumber(), e);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String getQuery = "SELECT * FROM drivers WHERE id = (?) AND is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                        PreparedStatement statement = connection.prepareStatement(getQuery)) {
+            statement.setObject(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(getDriver(resultSet));
+            }
+            return Optional.empty();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get driver by id : " + id, e);
+        }
+    }
+
+    @Override
+    public List<Driver> getAll() {
+        String getAllQuery = "SELECT * FROM drivers WHERE is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(getAllQuery)) {
+            List<Driver> allDrivers = new ArrayList<>();
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                allDrivers.add(getDriver(resultSet));
+            }
+            return allDrivers;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all drivers", e);
+        }
+    }
+
+    @Override
+    public Driver update(Driver driver) {
+        String updateQuery = "UPDATE drivers SET name = ?, license_number = ? "
+                + "WHERE id = ? AND is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(updateQuery)) {
+            statement.setString(1, driver.getName());
+            statement.setString(2, driver.getLicenseNumber());
+            statement.setObject(3, driver.getId());
+            statement.executeUpdate();
+            return driver;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update data : id - " + driver.getId()
+            + " name - " + driver.getName()
+                    + " license_number - " + driver.getLicenseNumber(), e);
```

Comment:
можна просто передавати driver в якому перевизначено toString() і тоді не потрібно для кожного поля виводити інформацію. Адже це може бути не зручно, якщо в нас більше 10 полів
```suggestion
            throw new DataProcessingException("Can't update data : id - " + driver, e);
```